### PR TITLE
fix: filenames in schedule.tsv

### PR DESCRIPTION
**Pull request by esavary**
Fix the filenames in schedule.tsv 
Related to #394

### CODE REVIEW

**Review by acionca — APPROVED** [review]
Looks good, I am just not sure why you would modify the column names from `_events` to `_edf`

**Comment by esavary** [comment]
> Looks good, I am just not sure why you would modify the column names from `_events` to `_edf`

Yes, good catch that was not intentional.

File: code/events/schedule.tsv [hunk 1]
```diff
@@ -1,28 +1,38 @@
-session	day	PE	dwi_events	qct_events	rest_events	bht_events
-1	2023-10-20	LR	fixation_2023-10-20_18h46.06.106_4_session_1.log	qct_2023-10-20_19h40.38.964_2_session_1.log	rest_2023-10-20_19h48.22.344_0_session_1.log	bht_2023-10-20_20h10.50.517_0_session_1.log
+session	day	PE	dwi_edf	qct_edf	rest_edf	bht_edf
```
Comment (esavary) [inline code/events/schedule.tsv]:
```suggestion
session	day	PE	dwi_events	qct_events	rest_events bht_events
```

File: code/events/schedule.tsv [hunk 2]
```diff
@@ -1,28 +1,38 @@
-session	day	PE	dwi_events	qct_events	rest_events	bht_events
-1	2023-10-20	LR	fixation_2023-10-20_18h46.06.106_4_session_1.log	qct_2023-10-20_19h40.38.964_2_session_1.log	rest_2023-10-20_19h48.22.344_0_session_1.log	bht_2023-10-20_20h10.50.517_0_session_1.log
+session	day	PE	dwi_events	qct_events	rest_events  bht_events
```
Comment (oesteban) [inline code/events/schedule.tsv]: This needs to remain a tab.
```suggestion
session	day	PE	dwi_events	qct_events	rest_events	bht_events
```

File: code/events/schedule.tsv [hunk 3]
```diff
@@ -1,28 +1,38 @@
...
+27	2023-11-02	AP	fixation_2023-11-02_19h29.37.833_1_session_27.log	n/a	n/a	n/a
 28	2023-11-02	RL	fixation_2023-11-02_21h17.30.822_1_session_28.log	qct_2023-11-02_21h55.55.757_0_session_28.log	rest_2023-11-02_22h03.31.967_0_session_28.log	bht_2023-11-02_22h25.06.651_0_session_28.log
+29	2023-11-03	PA	fixation_2023-11-03_19h15.36.495_2_session_29.log	qct_2023-11-03_19h58.22.540_0_session_29.log	rest_2023-11-03_20h06.12.290_0_session_29.log	bht_2023-11-03_20h27.56.353_0_session_29.log
+30		AP	n/a	n/a	n/a	n/a
+31		PA	n/a	n/a	n/a	n/a
+32		AP	n/a	n/a	n/a	n/a
+33		LR	n/a	n/a	n/a	n/a
+34		LR	n/a	n/a	n/a	n/a
+35		LR	n/a	n/a	n/a	n/a
+36		RL	n/a	n/a	n/a	n/a
```
Comment (oesteban) [inline code/events/schedule.tsv]: What's the point of this?
```suggestion
```

See https://github.com/TheAxonLab/hcph-sops/blob/mkdocs/code/events/schedule-pilot-excl.tsv

File: code/events/schedule.tsv [hunk 4]
```diff
@@ -1,28 +1,38 @@
...
 23	2023-10-31	AP	fixation_2023-10-31_19h38.18.771_1_session_23.log	qct_2023-10-31_20h16.23.124_0_session_23.log	rest_2023-10-31_20h23.08.716_0_session_23.log	bht_2023-10-31_20h45.02.641_0_session_23.log
-24	2023-10-31	AP	fixation_2023-10-31_21h38.40.916_1_session_24.log	qct_2023-10-31_22h29.32.491_0_session_24.log	rest_2023-10-31_22h36.04.198_0_session_24.log	bht_2023-10-31_22h58.13.607_0_session_24.log
+24	2023-10-31	AP	fixation_2023-10-31_21h49.38.233_2_session_24.log	qct_2023-10-31_22h29.32.491_0_session_24.log	rest_2023-10-31_22h36.04.198_0_session_24.log	bht_2023-10-31_22h58.13.607_0_session_24.log
 25	2023-11-01	RL	fixation_2023-11-01_19h34.54.237_1_session_25.log	qct_2023-11-01_20h13.26.444_0_session_25.log	rest_2023-11-01_20h19.46.432_0_session_25.log	bht_2023-11-01_20h41.25.152_0_session_25.log
 26	2023-11-01	PA	fixation_2023-11-01_21h19.06.608_1_session_26.log	qct_2023-11-01_21h55.19.742_0_session_26.log	rest_2023-11-01_22h03.59.490_0_session_26.log	bht_2023-11-01_22h26.03.086_0_session_26.log
+27	2023-11-02	AP	fixation_2023-11-02_19h29.37.833_1_session_27.log	n/a	n/a	n/a
```
Comment (oesteban) [inline code/events/schedule.tsv]:
```suggestion
```

https://github.com/TheAxonLab/hcph-sops/blob/mkdocs/code/events/schedule-pilot-excl.tsv

File: code/events/schedule.tsv [hunk 5]
```diff
@@ -1,28 +1,38 @@
...
+9	2023-10-24	AP	fixation_2023-10-24_19h39.27.478_3_session_9.log	qct_2023-10-24_20h15.40.693_0_session_9.log	rest_2023-10-24_20h24.27.482_0_session_9.log	bht_2023-10-24_20h46.08.308_0_session_9.log
 10	2023-10-24	RL	fixation_2023-10-24_21h26.14.900_1_session_10.log	qct_2023-10-24_22h10.40.236_0_session_10.log	rest_2023-10-24_22h18.14.132_0_session_10.log	bht_2023-10-24_22h41.05.920_0_session_10.log
 11	2023-10-25	AP	fixation_2023-10-25_20h37.34.053_7_session_11.log	qct_2023-10-25_21h12.18.840_0_session_11.log	rest_2023-10-25_21h23.26.254_0_session_11.log	bht_2023-10-25_21h47.06.131_1_session_11.log
 13	2023-10-26	PA	fixation_2023-10-26_19h27.13.413_3_session_13.log	qct_2023-10-26_20h05.24.294_0_session_13.log	rest_2023-10-26_20h13.47.119_1_session_13.log	bht_2023-10-26_20h36.26.995_1_session_13.log
-15	2023-10-27	AP	fixation_2023-10-27_19h21.04.397_0_session_15.log	qct_2023-10-27_19h59.03.664_0_session_15.log	rest_2023-10-27_20h06.19.329_0_session_15.log	bht_2023-10-27_20h31.31.806_0_session_15.log
+14	2023-10-26	LR	n/a	n/a	n/a	n/a
```
Comment (oesteban) [inline code/events/schedule.tsv]:
```suggestion
```

**Review by oesteban — CHANGES_REQUESTED** [review]
We need to be careful when changing things. The goal of this PR, if addressing #394, should be limited to only those changes. That means:

- no unintentional errors such as the change of the tab in the header with spaces
- no addition of lines that were removed in other PRs for a reason. If you believe these lines should be added, then open another PR to drive the corresponding conversation. In this case, however, those lines were intentionally removed previously and moved to a different file (https://github.com/TheAxonLab/hcph-sops/blob/mkdocs/code/events/schedule-pilot-excl.tsv).

**Comment by oesteban** [comment]
I guess this was closed by mistake?

**Comment by esavary** [comment]
> I guess this was closed by mistake?

Yes, thanks for reopening

**Pull request merged**